### ISSUE TITLE
AV: media player has no figure HTML

### ISSUE THREAD

**Issue by lizcameron** [bbc/simorgh#4647]
**Describe the bug**
The media player currently contains a figcaption but this is not contained within figure HTML. This means that items do not get announced as a group, affecting screenreader UX.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo
2. Scroll to the media player
3. Inspect
4. See figcaption is present without figure HTML:
`<figcaption class="Caption-sc-16x70so-0 gDfBZX"><span class="VisuallyHiddenText-sc-1yjwwa5-0 jFfEvX">Video caption, </span><p>This is a video block containing a video about ants</p></figcaption>`

**Expected behaviour**
Fig captions should be contained within figure HTML, as per the image with caption on the same test page (https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo)

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- This bug fix is expected to need manual testing.

**Additional context**
